sympy/functions/elementary/trigonometric.py

```diff
@@ -1948,6 +1948,26 @@ def eval(cls, arg):
         if i_coeff is not None:
             return S.ImaginaryUnit * asinh(i_coeff)
 
+        if isinstance(arg, sin):
+            ang = arg.args[0]
+            if not ang.has(Symbol) and ang.is_real:
+                ang %= 2*pi # restrict to [0,2*pi)
+                if ang > pi: # restrict to (-pi,pi]
+                    ang = pi-ang
```

Comment:
In SymPy code, there are usually spaces on both sides of + and - operators.

Reply:
Yes, thank you, added spaces.
@normalhuman Please review.